robotoff/insights/question.py

```diff
@@ -164,6 +170,67 @@ def get_source_image_url(barcode: str) -> Optional[str]:
 
         return None
 
+    @staticmethod
+    def generate_selected_images(images: JSONType, barcode: str) -> JSONType:
+        selected_images: Dict[str, Dict[str, Dict[str, str]]] = {
+            "front": {"display": {}, "small": {}, "thumb": {}},
+            "nutrition": {"display": {}, "small": {}, "thumb": {}},
+            "ingredients": {"display": {}, "small": {}, "thumb": {}},
+            "packaging": {"display": {}, "small": {}, "thumb": {}},
+        }
+        splitted_barcode = split_barcode(barcode)  # splitting the barcode
```

Comment:
As mentioned in an other comment, there is a `generate_image_url` function that does exactly this, you should use it